src/python/gudhi/simplex_tree.pyx

```diff
@@ -63,6 +81,25 @@ cdef class SimplexTree:
          """
         return self.pcohptr != NULL
 
+    def copy(self):
+        """ 
+        :returns: A simplex tree that is a deep copy of itself.
+        :rtype: SimplexTree
+
+        :note: copy requires :func:`compute_persistence` to be launched again as the persistence result is not copied.
```

Comment:
The formulation looks a bit too strong, people may use the SimplexTree for other purposes and never care about persistence. Maybe "the persistence information is not copied. If you need it in the clone, you have to call `compute_persistence` on it even if you had already computed it in the original."?

Reply:
Re-formulated on e7ba896